schemas/deviceToCloud/gps/gps.json

```diff
@@ -1,58 +1,28 @@
 {
-    "title":"GPS",
-    "description":"GPS device coordinate data",
-    "type":"object",
-    "oneOf":[
-       {
-          "properties":{
-             "appId":{
-                "type":"string",
-                "const":"GPS"
-             },
-             "messageType":{
-                "type":"string",
-                "const":"HELLO"
-             },
-            "ts": {
-                "type": "number",
-                "description": "Unix timestamp given in milliseconds when the data was sampled"
-            },
-            "time": {
-                "type": "number",
-                "description": "The same as the 'ts' property. Used for backwards compatibility for some firmware versions of ATv2. Please use the 'ts' field instead"
-                }
-            },
-          "additionalProperties":false
-       },
-       {
-          "properties":{
-             "appId":{
-                "type":"string",
-                "const":"GPS"
-             },
-             "messageType":{
-                "type":"string",
-                "const":"DATA"
-             },
-             "data":{
-                "type":"string",
-                "description":"NEMA string",
-                "pattern":"^\\$(GPGGA|GPGLL|GPGSA|GPGSV|GPRMC|GPVTG)(,[0-9A-Z\\*\\.]*)+\n?$"
-             },
-            "ts": {
-                "type": "number",
-                "description": "Unix timestamp given in milliseconds when the data was sampled"
-            },
-            "time": {
-                "type": "number",
-                "description": "The same as the 'ts' property. Used for backwards compatibility for some firmware versions of ATv2. Please use the 'ts' field instead"
-                }
-          },
-          "additionalProperties":false
-       }
-    ],
-    "required":[
-       "appId",
-       "messageType"
-    ]
- }
+   "title": "GPS",
```

Comment:
```suggestion
   "title": "GNSS",
```

We are switching to the `GNSS` term.

Reply:
Upon further investigation, it seems that the GPS HELLO message may not be used any more. I can't find a specific handler for it on the cloud and Pete says the same about the FW. Do you know if it's still used @coderbyheart ?

Reply:
I don't know.